.flake8

```diff
@@ -39,6 +39,28 @@ max-complexity = 10
 # Accessibility/large fonts and PEP8 friendly:
 max-line-length = 79
 
+# https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
+format = wemake
+
+# wemake-python-styleguide
+show-source = true
+
+# Allow certain violations in certain files:
+per-file-ignores =
+
+  # E800 reports a lot of false-positives for legit
+  # tool-related comments;
+  # WPS300 local folder imports are needed;
+  # WPS305 is unnecessary because this extension is
+  # Python 3 only and so f-strings are allowed;
+  # WPS317/WPS318 enforces weird indents;
+  # WPS326 doesn't allow implicit string concat;
```

Comment:
These entries should go to `extend-ignore` setting instead of being applied to every file separately.